enclave-tls/src/include/enclave-tls/api.h

```diff
@@ -41,16 +42,16 @@ typedef enum {
 typedef enum {
 	VERIFICATION_TYPE_QVL,
 	VERIFICATION_TYPE_QEL
-} quote_sgx_ecdsa_verification_type_t;
+} verifier_sgx_ecdsa_verification_type_t;
 // clang-format on
 
 typedef struct {
 	unsigned int api_version;
 	unsigned long flags;
 	enclave_tls_log_level_t log_level;
 	char tls_type[TLS_TYPE_NAME_SIZE];
-	char attester_type[QUOTE_TYPE_NAME_SIZE];
-	char verifier_type[QUOTE_TYPE_NAME_SIZE];
+	char attester_type[ATTESTER_TYPE_NAME_SIZE];
```

Comment:
Change to `ENCLAVE_ATTESTER_TYPE_NAME_SIZE`.

Reply:
done